### PR TITLE
Update GM-Commands.md

### PR DESCRIPTION
**Pull request by gengarshadowball**
## Description

Completes the PR https://github.com/azerothcore/azerothcore-wotlk/pull/2641
## Related Issue

Morph command update (merge)
## Motivation and Context

Documentation about how the morph command works from now on
## How Has This Been Tested?

compiled and tested in game

### CODE REVIEW

**Comment by BarbzYHOOL** [comment]
To merge when the PR is accepted

File: docs/GM-Commands.md [hunk 1]
```diff
@@ -491,7 +490,7 @@ Note: Some commands are working only by selecting a player or a creature. These
 |group disband|3|Syntax: .group disband [$characterName] Disbands the given character's group.|
 |group remove|3|Syntax: .group remove [$characterName] Removes the given character from his group.|
 |debug moveflags|3|Syntax: .debug moveflags [$newMoveFlags [$newMoveFlags2]] No params given will output the current moveflags of the target|
-|morph|2|Syntax: .morph #displayid Change your current model id to #displayid.|
+|morph|2|Syntax: .morph #displayid (Change your current model id to #displayid.) or .morph reset (Reset the original model for the selected target)|
```
Comment (Kitzunu) [inline docs/GM-Commands.md]:
```suggestion
|morph|2|Syntax: .morph #displayid Change your current model id to #displayid.|
```

Comment (Kitzunu) [reply docs/GM-Commands.md]: and add:
```md
|morph reset|2|Syntax: .morph reset Reset the original model for the selected target
```
under to match the other already entered commands in the file

**Comment by BarbzYHOOL** [comment]
thank you my dear @gengarshadowball

**Pull request merged**